server/src/main/scala/slickdb/FeatureGuideTable.scala

```diff
@@ -27,7 +27,15 @@ class FeatureGuideTable(tag: Tag) extends Table[FeatureGuideRow](tag, "feature_g
     (id, uploadTime, fileName, title, markdownContent, published).mapTo[FeatureGuideRow]
 }
 
-object FeatureGuideRow {
+trait FeatureGuideRowTableLike {
```

Comment:
It probably should have both `Row` and `Table` in the name - is it representing a row or the table?